.eslintrc

```diff
@@ -1,4 +1,8 @@
 ---
-extends: '@elastic/kibana'
 rules:
   no-unused-vars: off
+plugins:
+  ['react']
+extends:
+  ['@elastic/kibana',
+   'plugin:react/recommended']
```

Comment:
@kjbekkelund added the react eslint here.  Do you suggest the recommended list? or should I add more?

 @spalger am I adding these rules the correct way?